Description: Fix bash path detection on merged usr
Author: Andreas Metzler <ametzler@debian.org>
Origin: vendor
Forwarded: no
Last-Update: 2022-11-15

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -61,11 +61,11 @@
 endif( HAS_BRANCH_PREDICTION )
 
 # ======== Find bash =======
 
 # TODO: What if the check fails
-find_program (BASH_EXECUTABLE bash)
+find_program (BASH_EXECUTABLE bash PATHS /bin /usr/bin NO_DEFAULT_PATH)
 message( "Using bash: ${BASH_EXECUTABLE}" )
 
 # ======== Find perl =======
 
 find_program (PERL_EXECUTABLE perl)
